.travis.d/definition_check.py

```diff
@@ -80,7 +80,7 @@ def read_nids():
                         nids = user_nids
                 if SECTION != 'functions':
                     continue
-                if nids.get(k):
+                if nids != user_nids and nids.get(k):
```

Comment:
is it safe to check ther user_nids?

Reply:
I think no.
Or list only really common functions and allow skipping only if the target function is duplicated on list.
In here, just memcpy/memset to list.

Reply:
Added a patch to checks duplicate names for each module.